platform_plugin_teams/lms/api/views.py

```diff
@@ -0,0 +1,342 @@
+""" This file contains the API views for the Teams plugin. """ ""
```

Comment:
It seems to me that it would be an excellent option, I will check it out.

Reply:
I am trying to apply these changes and leave everything in a single view, however, it is failing me because of the LMS/CMS imports.

If I try to consume an LMS endpoint, having CMS imports generates an error. The same thing happens when I consume a CMS endpoint, having LMS imports (several) generates an error.

For example, I consume the DELETE topics endpoint and I get this error:
```bash
tutor_nightly_dev-cms-1  |   File "/openedx/requirements/platform-plugin-teams/platform_plugin_teams/urls.py", line 9, in <module>
tutor_nightly_dev-cms-1  |     include(
tutor_nightly_dev-cms-1  |   File "/openedx/venv/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
tutor_nightly_dev-cms-1  |     urlconf_module = import_module(urlconf_module)
tutor_nightly_dev-cms-1  |   File "/opt/pyenv/versions/3.8.15/lib/python3.8/importlib/__init__.py", line 127, in import_module
tutor_nightly_dev-cms-1  |     return _bootstrap._gcd_import(name[level:], package, level)
tutor_nightly_dev-cms-1  |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
tutor_nightly_dev-cms-1  |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
tutor_nightly_dev-cms-1  |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
tutor_nightly_dev-cms-1  |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
tutor_nightly_dev-cms-1  |   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
tutor_nightly_dev-cms-1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
tutor_nightly_dev-cms-1  |   File "/openedx/requirements/platform-plugin-teams/platform_plugin_teams/api/urls.py", line 5, in <module>
tutor_nightly_dev-cms-1  |     from platform_plugin_teams.api import views
tutor_nightly_dev-cms-1  |   File "/openedx/requirements/platform-plugin-teams/platform_plugin_teams/api/views.py", line 21, in <module>
tutor_nightly_dev-cms-1  |     from lms.djangoapps.teams.serializers import MembershipSerializer
tutor_nightly_dev-cms-1  |   File "/openedx/edx-platform/lms/djangoapps/teams/serializers.py", line 48, in <module>
tutor_nightly_dev-cms-1  |     class UserMembershipSerializer(serializers.ModelSerializer):
tutor_nightly_dev-cms-1  |   File "/openedx/edx-platform/lms/djangoapps/teams/serializers.py", line 53, in UserMembershipSerializer
tutor_nightly_dev-cms-1  |     profile_configuration = deepcopy(settings.ACCOUNT_VISIBILITY_CONFIGURATION)
tutor_nightly_dev-cms-1  |   File "/openedx/venv/lib/python3.8/site-packages/django/conf/__init__.py", line 83, in __getattr__
tutor_nightly_dev-cms-1  |     val = getattr(self._wrapped, name)
tutor_nightly_dev-cms-1  | AttributeError: 'Settings' object has no attribute 'ACCOUNT_VISIBILITY_CONFIGURATION'
```
But after removing the imports from the LMS it works without any problems.

Reply:
Let's leave it as it is then. Thanks!